hass-configurator/config.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "HASS Configurator",
+    "version": "0.1.2",
+    "slug": "hass-configurator",
+    "description": "HASS Configurator - Browser-based configuration file editor for Home Assistant.",
+    "url": "https://github.com/danielperna84/hassio-addons/tree/master/hass-configurator",
+    "startup": "application",
+    "webui": "http://[HOST]:[PORT:3218]",
+    "boot": "auto",
+    "ports": {
+      "3218/tcp": 3218
+    },
+    "map": ["config:rw", "ssl"],
+    "options": {
+      "BASEPATH": "/config",
+      "HASS_API": "http://172.17.0.1:8123/api/",
```

Comment:
hassio have a own network so you can use names:
`http://homeassistant:8123/api/`